src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -172,6 +172,7 @@ void CompiledModel::set_property(const ov::AnyMap& properties) {
                        ov::auto_batch_timeout.name());
```

Comment:
`if (time_out == properties.end() || properties.size() > 1) {`
Should we first of all set the known plugin properties and after that throw an exception if properties contains some unknown properties?